<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gst_scrollbar_app</class>
 <widget class="QMainWindow" name="gst_scrollbar_app">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gst_scrollbar_app</string>
  </property>
  <widget class="QWidget" name="main_window">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="add_camera_button">
            <property name="text">
             <string>Add Camera</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="rm_camera_button">
            <property name="text">
             <string>Remove Camera</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="start_rec_button">
            <property name="text">
             <string>Start Recording</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stop_rec_button">
            <property name="text">
             <string>Stop Recording</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Horizontal_Scroll_Area" name="Gst_Frame_Scroll">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>572</width>
             <height>245</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="debug_label">
        <property name="text">
         <string>Debug_Output</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="Debug_out_textbox"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>723</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Horizontal_Scroll_Area</class>
   <extends>QScrollArea</extends>
   <header>horizontal_scroll_area.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
